ConfigApp/Effects.cs

```diff
@@ -305,6 +305,7 @@ public enum EffectType
             EFFECT_MISC_FPS_LIMIT,
             EFFECT_META_NO_CHAOS,
             EFFECT_PEDS_ROASTING,
+			EFFECT_PLAYER_AIMBOT,
```

Comment:
These tabs should be spaces (same thing at the end of the file)

Reply:
what the hell am i reading

Reply:
i have no idea, why its misalligned here, its perfectly alligned in VS :|